Monika After Story/game/chess.rpy

```diff
@@ -3321,7 +3323,7 @@ init python:
                         store.mas_ptod.wx_cmd("import os", local_ctx)
                         renpy.pause(1.0)
                         store.mas_ptod.wx_cmd(
-                            "subprocess.call(['chmod','+x', os.path.normcase(basedir + '/game/mod_assets/games/chess/stockfish_8_{0}_x64')])".format(
+                            "subprocess.call(['chmod','+x', os.path.normcase(basedir + '/game/mod_assets/games/chess/stockfish_15_{0}_x64')])".format(
                                 "linux" if renpy.linux else "macosx"
```

Comment:
I'm not seeing a `stockfish_15_macosx_x64` binary. You're going to need the macos binary for this to work

Reply:
I do have a MacOS VM, I may be able to address your issue to download it

Reply:
Your offer of help is just what I need! I needed someone to help me with the MAC version of Stockfish.

As for Stockfish15, which has not been changed to MacOS in the code, it is to remind me that "the Mac problem is not solved."

Now that you say so, I'll get it fixed right away.

Reply:
Well, I mean, I can't really get the Mac version of Stockfish. Please help me with this download.
I just thought you meant there was no update in my code to allow MacOS to read Stockfish 15(Then I just noticed that the Linux address and the MacOS address change together)

Reply:
I'm working on it, Mac is just a pain to deal with lol